sqlmesh/core/plan/evaluator.py

```diff
@@ -203,7 +204,8 @@ def _backfill(
             interval_end_per_model=plan.interval_end_per_model,
         )
         if completion_status.is_failure:
-            raise PlanError("Plan application failed.")
+            self.console.log_status_update("[red]Error: Plan application failed.[/red]")
```

Comment:
Ditto. Rich specific formatting should be contained to the corresponding console implementation